build.gradle.kts

```diff
@@ -61,12 +61,8 @@ buildscript {
 plugins {
     `java-library`
     idea
-
-    val protobuf = io.spine.internal.dependency.Protobuf.GradlePlugin
```

Comment:
We cannot do it any more after detekt is added to classpath in `buildSrc/build.gradle.kts`.

Reply:
It gives compile error, saying `io` is uknown.